cypress/support/index.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
+
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import "./commands";
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
+
+// ignore uncaught exceptions
+Cypress.on("uncaught:exception", (err) => {
```

Comment:
would this catch errors generated from the app that are uncaught? or stuff specific to cypress only?

Reply:
This was a fallback for the wallet errors we were experiencing. I can probably remove this.

I think it was for uncaught exceptions that aren't necessarily causing cypress to fail but throw in the app.

Reply:
ok yea i think you could probably let it fail, or maybe catch and log it in a way we would be able to see it